Kong/docs.konghq.com
# Chore: Update plugin submodule

### Description

Updating the plugins submodule to fix 3.4 schemas and introduce 3.5.
This fixes the issue of incorrect nested descriptions in some plugins, for example OIDC:

[Before:](https://docs.konghq.com/hub/kong-inc/openid-connect/configuration/#config-session_remember_rolling_timeout)
![Screenshot 2023-10-26 at 10 30 14 AM](https://github.com/Kong/docs.konghq.com/assets/54370747/648b84ee-e1c6-4c1d-8dd0-d9fd10624214)

[After:](https://deploy-preview-6378--kongdocs.netlify.app/hub/kong-inc/openid-connect/configuration/#config-session_remember_rolling_timeout)
![Screenshot 2023-10-26 at 10 31 07 AM](https://github.com/Kong/docs.konghq.com/assets/54370747/da9de473-67bd-4a80-84bc-261c9a92488c)

3.5 won't show on the production/live site until the 3.5 release branch is merged, so there's no issue with pulling in 3.5 content here.

### Testing instructions

Preview link: Check that plugins with nested schema fields are rendering correctly, for example:
https://deploy-preview-6378--kongdocs.netlify.app/hub/kong-inc/openid-connect/configuration/#config-session_remember_rolling_timeout

### Checklist
